tests/integration/test_backup_restore_on_cluster/configs/disallow_concurrency.xml

```diff
@@ -0,0 +1,15 @@
+<clickhouse>
+    <storage_configuration>
+        <disks>
+            <backups>
+                <type>local</type>
+                <path>/backups/</path>
+            </backups>
+        </disks>
+    </storage_configuration>
+    <backups>
```

Comment:
The section `<backups>` was meant to organize all settings related to backups or restores.

Reply:
addressed in #45497